stylecop.ruleset

```diff
@@ -9,6 +9,11 @@
 	<Rule Id="SA1124" Action="None" />
 	<Rule Id="SA1129" Action="None" />
 	<Rule Id="SA1131" Action="None" />
+	<!-- 
```

Comment:
Please explain why changes. If you have build error about SA1135 in VS2019, please try to update to latest version.